src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Tov.hpp

```diff
@@ -60,8 +60,18 @@ class TovSolution {
   /// constructing this TovSolution
   double outer_radius() const noexcept;
 
+  /// \brief The mass inside the given radius over the radius
+  /// \f$\frac{m(r)}{r}\f$
+  ///
+  /// \note `r` should be non-negative and not greater than outer_radius
+  /// \f$m(R)\f$.
```

Comment:
As written, it's not obvious what "outer_radius" is or why you include the "m(R)" term. I suggest either making this simpler ("not greater than the radius of the star"), or expanding with more detail ("not greater than the radius of the star as returned by `outer_radius()`).

Reply:
I had just copied this from the `mass()` function, so adjusted the docs for that as well and squashed the docs change into the commit. The `outer_radius()` now properly links to the member function.